digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -874,7 +875,7 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
   put("/linkproperties") {
     val properties = parsedBody.extract[Seq[LinkProperties]]
     val user = userProvider.getCurrentUser()
-    def municipalityValidation(municipalityCode: Int, administrativeClass: AdministrativeClass) = validateUserAccess(user)(municipalityCode, administrativeClass)
+    def municipalityValidation(municipalityCode: Int, administrativeClass: AdministrativeClass) = validateUserAccess(user, 460)(municipalityCode, administrativeClass)
```

Comment:
Mitä toi 460 numero merkkaa?

Reply:
RoadLinkProperties type id. Selkeämpi olisi varmaan käyttää RoadLinkProperties.typeId

Reply:
Siinä tais tulla joku resolving error, kun tuo nimi on useammassa käyttötarkoituksessa.

Reply:
Pitäisi varmaan korjata eri nimet noille, mutta näin saa import jos nyt haluaa: 
import fi.liikennevirasto.digiroad2.asset.{RoadLinkProperties => RoadLinkPropertiesAsset}